BookReaderDemo/simple-bookreader-demo.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>BookReader Simple Demo</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+    <meta name="apple-mobile-web-app-capable" content="yes">
+
+    <!-- Get query string parameters and set title, if passed -->
+    <script>
+        let brDemoSearchParams = new URLSearchParams(window.location.search);
+
+        if (brDemoSearchParams.has('title')) {
+            document.title = brDemoSearchParams.get('title');
+        }
+    </script>
+
+    <!-- JS dependencies -->
+    <script src="../BookReader/jquery-1.10.1.js"></script>
+    <script src="../BookReader/jquery-ui-1.12.0.min.js"></script>
+    <script src="../BookReader/jquery.browser.min.js"></script>
+    <script src="../BookReader/dragscrollable-br.js"></script>
+    <script id = "last-js-dependency" src="../BookReader/jquery.colorbox-min.js"></script>
+    
+    <!-- Link to mmenu files, if needed for this demo -->
+    <script id="import-mmenu">
+        if (brDemoSearchParams.get('mmenu')) {
+            let mmenuScript = document.createElement('script');
+            mmenuScript.src = '../BookReader/mmenu/dist/js/jquery.mmenu.min.js';
+
+            let mmenuNavbarScript = document.createElement('script');
+            mmenuNavbarScript.src = '../BookReader/mmenu/dist/addons/navbars/jquery.mmenu.navbars.min.js';
+
+            $('#import-mmenu').after(
+                '<link rel="stylesheet" href="../BookReader/mmenu/dist/css/jquery.mmenu.css"/>',
+                '<link rel="stylesheet" href="../BookReader/mmenu/dist/addons/navbars/jquery.mmenu.navbars.css"/>',
+                mmenuScript,
+                mmenuNavbarScript
+            );
+        }
+    </script>
+
+    <!-- BookReader and plugins -->
+    <link id="bookreader-css" rel="stylesheet" href="../BookReader/BookReader.css"/>
+    <script src="../BookReader/BookReader.js"></script>
+
+    <!-- Link to plugins that were passed in the query string -->
+    <script id="import-plugins">
```

Comment:
The suggested `document.write()` approach wrote directly to the body, so I went with this approach.